src/utils/display.ts

```diff
@@ -97,24 +104,42 @@ export const makeSignOperationContent = (
         break;
       }
       case "UniswapV3 Swap": {
-        const { tokenIn, inAmount: amountSmallestUnits, tokenOut } = item;
+        const {
+          tokenIn,
+          inAmount: amountSmallestUnits,
+          tokenOut,
+          maxSlippageBps,
+          exactQuoteWei,
+          minimumAmountOutWei,
+        } = item;
         const tickerIn = lookupTickerByAddress(tokenIn, erc20s);
         const tickerOut = lookupTickerByAddress(tokenOut, erc20s);
-        const displayAmountIn = formatUnits(amountSmallestUnits);
+        const displayAmountIn = displayAmount(formatUnits(amountSmallestUnits));
+        const displayExactQuote = displayAmount(formatUnits(exactQuoteWei));
+        const displayMinimumAmountOut = displayAmount(
+          formatUnits(minimumAmountOutWei)
+        );
         heading = "Confirm token swap";
 
         if (tickerIn && tickerOut) {
           messages.push(
-            `Action: Swap **${displayAmountIn} ${tickerIn}** for **${tickerOut}**`
+            `Action: Swap **${displayAmountIn} ${tickerIn}** for ≈**${displayExactQuote} ${tickerOut}**`
           );
         } else {
           messages.push(
             "Action: Swap",
-            `Amount: **${displayAmountIn}**`,
             `From token: **${tickerIn ?? displayUnrecognizedAsset(tokenIn)}**`,
-            `To token: **${tickerOut ?? displayUnrecognizedAsset(tokenOut)}**`
+            `Amount: **${displayAmountIn}**`,
+            `To token: **${tickerOut ?? displayUnrecognizedAsset(tokenOut)}**`,
+            `Amount: ~**${displayExactQuote}**`
           );
         }
+        messages.push(
+          `Max slippage: ${maxSlippageBps / 100}%`,
```

Comment:
do we want to wrap `maxSlippageBps` in `displayAmount` to limit sigfigs or nah?

Reply:
sure, done